serial: "LHR-09DF88FD"
frame: "dev"
gravity:          [0.0, 0.0, -9.80665]
rate: 128
initial_estimate:
  position:       [0, 0, 0]
  attitude:       [0, 0, 0, 1]
  velocity:       [0, 0, 0]
  acceleration:   [0, 0, 0]
  omega:          [0, 0, 0]
  gyro_bias:      [0, 0, 0]
initial_covariance:
  position:       [1, 1, 1]
  attitude:       [10, 10, 10]
  velocity:       [1.0e-06, 1.0e-06, 1.0e-06]
  acceleration:   [1.0e-06, 1.0e-06, 1.0e-06]
  omega:          [1.0e-06, 1.0e-06, 1.0e-06]
  gyro_bias:      [1.0e-01, 1.0e-01, 1.0e-01]
process_noise:
  position:       [1.0e-06, 1.0e-06, 1.0e-06]
  attitude:       [1.0e-02, 1.0e-02, 1.0e-02]
  velocity:       [1.0e-08, 1.0e-08, 1.0e-08]
  acceleration:   [1.0e-10, 1.0e-10, 1.0e-10]
  omega:          [1.0e-08, 1.0e-08, 1.0e-08]
  gyro_bias:      [2.4e-10, 2.4e-10, 2.4e-10]